### PR TITLE
my JS week2 home work

### PR DESCRIPTION
**Pull request by husenjanobul**
this is js1 week2 home work .
the week1 home work was ,  we have done in the class.

### CODE REVIEW

File: Week2/arrays.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var myArrys = [];
+console.log("arrays are used to store multiple values in a single variable.");
```
Comment (marcianoviereck) [inline Week2/arrays.js:3]: yeah that is correct, but what do you think the value of the `myArrs`?

File: Week2/arrays.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var myArrys = [];
```
Comment (marcianoviereck) [inline Week2/arrays.js]: It's better to use `let` or `const` instead of `var`, so can you change that:)?

File: Week2/assign-string.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+console.log("string is zero or more characters written inside quotes.");
+var y = "huseyin";
+console.log(y);
```
Comment (marcianoviereck) [inline Week2/assign-string.js:4]: Ah, here again, your explanation is correct, but waht do you think the value of `y` is?

File: Week2/assign-string.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+console.log("string is zero or more characters written inside quotes.");
+var y = "huseyin";
```
Comment (marcianoviereck) [inline Week2/assign-string.js]: Again the same comment with the `var` :)

File: Week2/round-number.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+console.log(z);
+var a = 7;
+console.log(a);
+console.log("the highest value is : 7.25");
+
+// second
+var a = Math.round(z);
+console.log(a);
+var c = z - a;
+if ( c > 0 && z != a ){
```
Comment (marcianoviereck) [inline Week2/round-number.js]: Ah wow, cool solution!
There is a much easier way to do this, you can use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max) it makes this a lot easier

**Review by marcianoviereck — COMMENTED** [review]
The homework looks really nice and some solutions are pretty cool! But can you change the `var`'s and look at my other comments?:)

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
Forgot to mark this PR as 'request changes'

File: Week2/arrays.js (hunk 1 above)
Comment (husenjanobul) [reply Week2/arrays.js:3]: now i made it .

File: Week2/arrays.js (hunk 2 above)
Comment (husenjanobul) [reply Week2/arrays.js]: thank you, i changed var to let.

File: Week2/round-number.js (hunk 5 above)
Comment (husenjanobul) [reply Week2/round-number.js]: i will try this.

Comment (husenjanobul) [reply Week2/round-number.js]: @marcianoviereck  thanks for feedback.

File: Week2/round-number.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
```
Comment (marcianoviereck) [inline Week2/round-number.js]: Ah your new solution (on line 18) is fine, but can you remove this code then?

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The homework is almost approved, can you only remove the old code in the `round-number.js`?

File: Week2/declare-integer.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+var x;
```
Comment (marcianoviereck) [inline Week2/declare-integer.js]: Can you change this `var` to `let` or `const`?

File: Week2/round-number.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+var z = 7.25;
```
Comment (marcianoviereck) [inline Week2/round-number.js]: Can you change this `var` to `let` or `const`?

File: Week2/round-number.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+console.log(z);
+var a = 7;
```
Comment (marcianoviereck) [inline Week2/round-number.js]: Can you change this `var` to `let` or `const`?

File: Week2/round-number.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+console.log(a);
+console.log("the highest value is : 7.25");
+
+// second
+var a = Math.round(z);
```
Comment (marcianoviereck) [inline Week2/round-number.js]: Can you change this `var` to `let` or `const`?

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
And can you also cahnge your other `var`s to `let` or `const`?

**Review by marcianoviereck — APPROVED** [review]
Ah nice! The homework is approved now!

**Pull request closed without merging**